.github/workflows/macos_build_hotfix_release.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     name: Run Tests
 
     needs: assert_release_branch
-    uses: ./.github/workflows/pr.yml
+    uses: ./.github/workflows/macos_pr.yml
```

Comment:
An inconsistency between iOS and macOS is that iOS uses hyphens for its workflow names, and macOS uses underscores. In hindsight I'd prefer to change iOS to also use underscores, I'd propose doing that during the monorepo transition.